cmd/server/main.go

```diff
@@ -85,6 +104,7 @@ func init() {
 	flag.StringVar(&Address, "address", config.SERVER_ADDRESS, "the gRPC server address")
 	flag.StringVar(&HealthAddress, "health", "0.0.0.0:8089", "Address for health checking")
 	flag.StringVar(&Dsn, "dsn", "", "")
+	flag.StringVar(&PDPAddr, "pdp", "", "address of the pdp")
```

Comment:
Awesome, but please call this `authz` or `authz-service`.

Also, after this please pick up https://github.com/infobloxopen/atlas-app-toolkit/pull/6 and add this to the flags package there, and migrate this server to use the packages in the PR.

Reply:
> Awesome, but please call this authz or authz-service.

Will do!

> Also, after this please pick up infobloxopen/atlas-app-toolkit#6 and add this to the flags package there, and migrate this server to use the packages in the PR.

Sounds good. I'm planning to bring this up at tomorrow's sprint planning — it's definitely something I'd be interested to take on.